Makefile

```diff
@@ -85,6 +88,11 @@ else
 	BUILDFLAGS := -buildmode=pie
 endif
 
+# whether stipping the binary
+ifeq ($(STRIP),yes)
+	LDFLAGS += -w -s
```

Comment:
Could you rename it with KATA_LDFLAGS LDFLAGS empty will use the ones from the host and some of them are not compatible with golang. In the agent may not be a big issue as we build it in a container based in recent experience could be  better if yes, will need to change https://github.com/kata-containers/agent/pull/740/files#diff-b67911656ef5d18c4ae36cb6741b7965R105  to use it